beacon_node/beacon_processor/src/rayon_manager.rs

```diff
@@ -0,0 +1,37 @@
+use rayon::{ThreadPool, ThreadPoolBuilder};
+use std::sync::Arc;
+
+const DEFAULT_LOW_PRIORITY_DIVISOR: usize = 4;
+const MINIMUM_LOW_PRIORITY_THREAD_COUNT: usize = 2;
+
+pub struct RayonManager {
+    /// Smaller rayon thread pool for lower-priority, compute-intensive tasks.
+    /// By default ~25% of CPUs or a minimum of 2 threads.
+    pub low_priority_threadpool: Arc<ThreadPool>,
+    /// Larger rayon thread pool for high-priority, compute-intensive tasks.
+    /// By default 100% of CPUs.
+    pub high_priority_threadpool: Arc<ThreadPool>,
```

Comment:
I noticed that we're no longer using `high_priority_threadpool`, should we remove it?